src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurEventNotificationApiServiceImpl.java

```diff
@@ -26,11 +27,10 @@
 import edu.cornell.kfs.concur.batch.ConcurWebRequest;
 import edu.cornell.kfs.concur.batch.ConcurWebRequestBuilder;
 import edu.cornell.kfs.concur.batch.service.ConcurBatchUtilityService;
-import edu.cornell.kfs.concur.batch.service.ConcurEventNotificationV2WebserviceService;
 import edu.cornell.kfs.sys.util.CUJsonUtils;
 import edu.cornell.kfs.sys.util.CURestClientUtils;
 
-public class ConcurEventNotificationV2WebserviceServiceImpl implements ConcurEventNotificationV2WebserviceService {
+public class ConcurEventNotificationApiServiceImpl implements ConcurEventNotificationApiService {
```

Comment:
Since this service is used to call Concur webservice endpoints, I think it should have Webservice in the name